libraries/native/native/eosio/crt.hpp

```diff
@@ -8,7 +8,7 @@ namespace eosio { namespace cdt {
       none
    };
    struct output_stream {
-      char output[1024*2];
+      char output[1024*4];
```

Comment:
I understand that this fixes the bus error in the unit test. However, increasing the buffer size fixes a symptom rather than the underlying problem. The current `output_stream` implementation is a disaster waiting to happen (again). We need to either fix the implementation or at least put safeguards to check for buffer overruns.